x/evm/keeper/funtoken_from_coin.go

```diff
@@ -114,7 +114,5 @@ func (k *Keeper) deployERC20ForBankCoin(
 		return gethcommon.Address{}, errors.Wrap(err, "failed to commit stateDB")
 	}
 
-	ctx.GasMeter().ConsumeGas(evmResp.GasUsed, "deploy erc20 funtoken contract")
```

Comment:
We no longer need to consume gas here since we consume it at a lower level in `CallContractWithInput()`.